plugins.json

```diff
@@ -4475,6 +4475,93 @@
+    "download_url": "https://plugins-store.test.mattermost.com/release/mattermost-plugin-msteams-meetings-v2.0.0.tar.gz",
+    "release_notes_url": "https://github.com/mattermost/mattermost-plugin-msteams-meetings/releases/tag/v2.0.0",
+    "hosting": "",
+    "author_type": "mattermost",
+    "release_stage": "production",
+    "enterprise": false,
```

Comment:
Should this be true?

Reply:
MS team meetings is available in our Professional and Enterprise packages (not Starter). Same for Cloud & Self-hosted

Reply:
THanks!